src/nimblepkg/version.nim

```diff
@@ -29,6 +29,9 @@ type
     of verAny:
       nil
 
+  ## Tuple containing package name and version range.
+  PkgTuple* = tuple[name: string, ver: VersionRange]
```

Comment:
Perhaps not necessary now, but necessary for the upcoming PRs. Also I think it does make sense here too.